### ISSUE TITLE
A11y_SQL Azure Data Studio_Notebooks_Python Dependencies Wizard_ColorContrast: color contrast ratio is less than 4.5:1 for selected dropdown list items.

### ISSUE THREAD

**Issue by SachinChavhan317** [microsoft/azuredatastudio#15853]
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.3;#ColorContrast;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.30.0-insider (user setup)
Windows Version: Windows10

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Press the context menu key on connection and hit enter on New Notebook
4. Open the Command Palette by pressing CTRL+SHIFT+P (Windows) or CMD+SHIFT+P (Mac). Alternatively, you can select the gear icon at the bottom left of the screen and select “Command Palette…” from the menu that appears.
5. Type “Python” into the text box that appears. Select “Configure Python for Notebooks” from the menu. Python Dependencies Wizard will appear.
6. Provide any Input file details & Next.
7. Tab till install and press enter.
8. Open accessibility insight for windows and start the color contrast option.
9. Check the color contrast ratio is  4.15:1 for selected dropdown list items.

**Actual:**
color contrast ratio is  4.15:1 for selected dropdown list items.

Similar issue observed in: 
Import Extension - Summary- ColorContrast : color contrast ratio is less than 4.5:1 for selected dropdown list items.

**Expected:**
color contrast ratio should be greater than 4.5:1 for selected dropdown list items.

**User Impact:**
low vision users will face difficulty while accessing the control as color contrast ratio is less than 4.5:1 for selected dropdown list items.

**Comment by chlafreniere** [comment]
This comes from VS Code. Explanation can be found here: https://github.com/microsoft/azuredatastudio/issues/11196#issuecomment-687326376

**Comment by Raghu-addepalli** [comment]
![image](https://user-images.githubusercontent.com/91586743/218436205-33416bf3-a367-4016-8675-15a90a60f1a5.png)
verified bug in below version issue is fixed, hence closing the bug and added attachment.
Version: 1.41.2 (system setup)
Commit: 0f0960d0e774e1ce903b79378addff96117c27a6
Date: 2023-02-08T20:16:49.071Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

Git Hub Tags : #Closed;

**Comment by Raghu-addepalli** [comment]
![image](https://user-images.githubusercontent.com/91586743/218438895-ae146dc9-134a-4921-904d-029ae1b488e2.png)
Version: 1.42.0-insider (user setup)
Commit: 99a924dbcdce7bbaa0ab78b19f84e83035e54651
Date: 2023-02-08T06:58:10.882Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623


Git Hub Tags: #Closed;
